### PR TITLE
add trajan package

### PR DESCRIPTION
**Pull request by gauteh**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [ x Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by gauteh** [comment]
@conda-forge/help-python, ready for review!

File: recipes/trajan/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a7b44ccd9898f45ac04033e46158f01748962ba9a943329ba0c0e27b59743296
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - trajanshow = trajan.scripts.trajanshow:main
```
Comment (synapticarbors) [inline recipes/trajan/meta.yaml:17]: Please add a test for the entry point:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands

File: recipes/trajan/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - poetry
+  run:
```
Comment (synapticarbors) [inline recipes/trajan/meta.yaml:24]: Many of these dependencies have minimum versions in the upstream `setup.py`/`pyproject.toml`. We generally would like to see matching restrictions here.

File: recipes/trajan/LICENSE [hunk 3]
```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (synapticarbors) [inline recipes/trajan/LICENSE:1]: You do not need to include this copy of the license file since it is included in the source distribution.

**Review by synapticarbors — CHANGES_REQUESTED** [review]

**Comment by knutfrode** [comment]
I agree to be maintainer for this package

File: recipes/trajan/LICENSE (hunk 3 above)
Comment (gauteh) [reply recipes/trajan/LICENSE:1]: Seems to be necessary, according to linter.

File: recipes/trajan/meta.yaml (hunk 2 above)
Comment (gauteh) [reply recipes/trajan/meta.yaml:24]: Done.

File: recipes/trajan/meta.yaml (hunk 1 above)
Comment (gauteh) [reply recipes/trajan/meta.yaml:17]: Done.

File: recipes/trajan/LICENSE (hunk 3 above)
Comment (synapticarbors) [reply recipes/trajan/LICENSE:1]: You should be able to remove the file, but you would need to keep `license_file: LICENSE` in the `meta.yaml`

Comment (gauteh) [reply recipes/trajan/LICENSE]: Ah ok, thanks.

File: recipes/trajan/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  host:
+    - python >=3.8
+    - pip
+    - poetry >=1
+  run:
+    - python >=3.8
+    - numpy >=1.23
+    - scipy >=1.9
+    - xarray >=2022.6.0
+    - matplotlib-base >=3.8
```
[the remark this answers is not in the document]
Comment (gauteh) [reply recipes/trajan/meta.yaml]: Nice catch, fixed.

**Comment by gauteh** [comment]
Thank you!

**Pull request merged**